_data/toc-mapping.yml

```diff
@@ -0,0 +1,6 @@
+# This file can be used to explicitly map a release to a specific table-of-contents
```

Comment:
Can we place the versioning data (e.g., the toc and the mapping) in a subfolder of data? Perhaps version/ or similar?

Reply:
I've just committed the changes that will allow this and updated the documentation too. The folder name is configurable via the _config.yml file.